diff --git src/utils_mount.c src/utils_mount.c
index b8dabb1..fc122a6 100644
--- src/utils_mount.c
+++ src/utils_mount.c
@@ -24,17 +24,16 @@
 # include "config.h"
 #endif
 
-#if HAVE_XFS_XQM_H
 # define _GNU_SOURCE
+#include "collectd.h"
+#include "utils_mount.h"
+
+#if HAVE_XFS_XQM_H
 # include <xfs/xqm.h>
 #define XFS_SUPER_MAGIC_STR "XFSB"
 #define XFS_SUPER_MAGIC2_STR "BSFX"
 #endif
 
-#include "collectd.h"
-
-#include "utils_mount.h"
-
 #include "common.h" /* sstrncpy() et alii */
 #include "plugin.h" /* ERROR() macro */
 
